main.js

```diff
@@ -2,14 +2,19 @@ const { join } = require('node:path');
 const { readFileSync } = require('node:fs');
 
 const lockfile = join(process.env.GITHUB_WORKSPACE, 'package-lock.json');
-const matcher = /.+\/(?<repo>[a-z-]+\/[a-z-]+)\.git#(?<hash>[0-9a-f]+)/;
+const matcher = /.+\/(?<repo>[\w-]+\/[a-z-]+)\.git#(?<hash>[0-9a-f]+)/;
```

Comment:
```suggestion
const matcher = /.+\/(?<repo>[\w-]+\/[\w-]+)\.git#(?<hash>[0-9a-f]+)/;
```
Should also change this one.

Reply:
maybe. I thought that it was supposed to enforce the `tree-sitter-xxxx` convention. I updated it

Reply:
updated it.